Covinator/Covinator/Pages/PfizerVaccination.cshtml.cs

```diff
@@ -59,7 +59,7 @@ public void OnGet()
                     foreach (string evt in validationEvents)
                     {
                         Console.WriteLine(evt);
-                        ViewData["PfizerVaccineDistributionAllocations"] = new PfizerVaccineDistributionAllocations();
+                        ViewData["PfizerVaccineDistributionAllocations"] = new PfizerVaccineDistributionAllocations []{ };
```

Comment:
Return empty array instead of generic empty object to avoid error.
Source:-
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions

Reply:
Good.